docs/src/administration/users.md

```diff
@@ -3,136 +3,188 @@ title: "User administration"
+You can redeploy by clicking **Redeploy** in the console.
 
-```bash
-$ git clone git@github.com:user/github-repo.git Project Name
-```
 
-both would error with
+### Change existing permissions for environment types
 
-```bash
-Failed to connect to the Git repository: git@github.com:user/github-repo.git
+To change the user permissions, follow these steps:
 
-Please make sure you have the correct access rights and the repository exists.
-```
+{{< codetabs >}}
+---
+title=In the console
+file=none
+highlight=false
+---
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+3. Select a user and change the permissions.
+4. Click **Save**.
+
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+Use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md)to change user permissions.
 
-despite their **Project Admin** access to the project.
+`platform user:update`
 
-This enhaces consistency over your source code, as the Platform.sh project functions as a read-only mirror of your remote repository. Otherwise, changes pushed directly to the project would be overwritten or deleted when commits are pushed via the integration. Platform.sh considers your integrated remote repository to be the "source of truth" as soon as it has been configured, and this caveat ensures that all commits go through the integration.
+  ```bash
+  platform user:update user1@example.com -r production:v,development:c
 
-The best course of action is to have your access updated on the integrated repository. If for some reason that is not a quick change, you can still clone through the project using the legacy pattern (which will set the *project* as its remote), but again, it is not recommended that you commit to the project once you have done so:
+  ```
 
-```bash
-$ git clone <project>@git.<region>.platform.sh:<project>.git
-```
+After you change a user's role for an environment type, you must trigger a redeploy each environment to propagate access changes. You can redeploy using the CLI command `platform redeploy`.
+
+{{< /codetabs >}}
+
+
+### Transfer project ownership
+
+You can transfer your plan ownership to a different [organization](/administration/organizations.md) anytime.
+You have to be the organization owner, or an organization user with [manage plan](/administration/organizations.md#organization-permissions) or [manage users](/administration/organizations.md#organization-permissions) permissions.
+
+1. Invite the new organization owner as a **Project Admin**.
+2. Submit a [support ticket](https://console.platform.sh/-/users/~/tickets) from your organization owner's account to ask for the transfer.
+
+Once the transfer is completed, the new organization can administer all project settings, billing, and gets the subscription charges.
```

Comment:
```suggestion
Once the transfer is completed, the new organization can administer all project settings and billing and receives future invoices.
```

Or I'm not sure what "subscription charges" are. I see invoices in my org, but I don't know if it's different for other tiers.  But "gets ... charges" seems confusing to me.